test/decl/enum/enumtest.swift

```diff
@@ -446,7 +446,7 @@ class C {}
 protocol P {}
 
 enum E : C & P {}
-// expected-error@-1 {{inheritance from class-constrained protocol composition type 'C & P'}}
+// expected-error@-1 {{Cannot inherit from class-constrained protocol composition type 'C & P'}}
```

Comment:
Nit: Lower cased as well. For all the occurrences =]
```suggestion
// expected-error@-1 {{cannot inherit from class-constrained protocol composition type 'C & P'}}
```